### PR TITLE
Ubuntu Fixes

### PR DESCRIPTION
**Pull request by markfeathers**
Fixed the packagelist for 22.04 to remove nonexistent package.
Fixed the host docker for ubuntu 23.04 to fix host gcc.

### CODE REVIEW

**Review by randytsi369 — APPROVED** [review]
Reviewed, not tested.

File: tasks/distros/ubuntu/lunar/host-armhf-docker/Dockerfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ RUN dpkg --add-architecture armhf
 RUN apt-get update
 # This must be aptitude because it will have to downgrade some packages
 # to install the armhf distribution
-RUN aptitude install -y libgpiod-dev:armhf libc6:armhf libgcc-s1:armhf libstdc++6:armhf gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf libinput-dev:armhf libiio-dev:armhf
+RUN aptitude install -y libgpiod-dev:armhf libc6-dev-armhf-cross gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf libinput-dev:armhf libiio-dev:armhf
```
Comment (ts-kris) [inline tasks/distros/ubuntu/lunar/host-armhf-docker/Dockerfile:23]: I'm assuming so, but, do the changed options still pull in the now removed `libstdc++6` for arm?

**Review by luckshot — DISMISSED** [review]
Approving pending discussed changes.

**Review by randytsi369 — APPROVED** [review]
Reviewed, not tested.

**Review by ts-kris — APPROVED** [review]
Looks good to me. Reviewed not tested

**Pull request merged**